src/common.jl

```diff
@@ -62,6 +69,11 @@ function SciMLBase.init(prob::LinearProblem, alg::Union{SciMLLinearSolveAlgorith
     A = alias_A ? A : deepcopy(A)
     b = alias_b ? b : deepcopy(b)
 
+
+    abstol   = (abstol   == 0) ? √eps(eltype(A)) : abstol
+    reltol   = (reltol   == 0) ? √eps(eltype(A)) : reltol
```

Comment:
where does this default come from?

Reply:
https://github.com/JuliaLinearAlgebra/IterativeSolvers.jl/blob/master/src/cg.jl#L122